src/AddLink.jsx

```diff
@@ -0,0 +1,19 @@
+import linkImg from './assets/link.svg';
```

Comment:
svg 파일이라서 -Icon으로 표현해주는 것도 괜찮을 것 같아요
```suggestion
import linkIcon from './assets/link.svg';
```

Reply:
아이콘은 img 태그로 사용해주기보다는 리액트 컴포넌트로 다뤄주시면 좋을 것 같아요.
[참고](https://www.designcise.com/web/tutorial/how-to-use-svg-as-component-with-create-react-app)